components/molecules/table/components/table.js

```diff
@@ -20,6 +27,8 @@ export const Table = ({
   className,
   columns,
   hover,
+  loading,
+  noData,
```

Comment:
@majiiid-drykiss We have to explicitly say noData? should it not be implied from the loading?

Reply:
Done